### PR TITLE
Changes in Account Requirements

### PR DESCRIPTION
**Pull request by joinimran**
As the user has reported and I have tested, we need to have a Work/School account required.

Problem: #2424

The old PR was mistakenly closed. Old PR link https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7100.

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by joinimran** [comment]
@e0i this is the new PR created against https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7100

File: windows/application-management/msix-app-packaging-tool.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ You can either run your installer interactively (through the UI) or create a pac
...
 - Windows 10, version 1809 (or later)
 - Participation in the Windows Insider Program (if you're using an Insider build)
-- A valid Microsoft account (MSA) alias to access the app from the Microsoft Store 
+- A valid Microsoft work or school account to access the app from the Microsoft Store 
 - Admin privileges on your PC account 
 
 ### Get the app from the Microsoft Store
 
-1. Use the MSA login associated with your Windows Insider Program credentials in the [Microsoft Store](https://www.microsoft.com/store/r/9N5LW3JBCXKF). 
+1. Use the work of school account login associated with your Windows Insider Program credentials in the [Microsoft Store](https://www.microsoft.com/store/r/9N5LW3JBCXKF).
```
Comment (JohanFreelancer9) [inline windows/application-management/msix-app-packaging-tool.md:38]:
```suggestion
1. Use the Microsoft work of school account login associated with your Windows Insider Program credentials in the [Microsoft Store](https://www.microsoft.com/store/r/9N5LW3JBCXKF). 
```

**Comment by joinimran** [comment]
Ready to go live @e0i

**Pull request merged**